src/main/pages/che-7/installation-guide/proc_using-che-with-tls.adoc

```diff
@@ -39,10 +39,12 @@ NOTE: Depending on the user’s OS, the *Export* (Linux-based OS) or *Save to Fi
 .Procedure
 
 . On the left side of the Chrome browser URL address bar panel, click the icon of the `Lock`, which represents the SSL/TLS status.
++
+This icon indicates the information about your connection.
```

Comment:
```suggestion
. On the left side of the Chrome browser URL address bar panel, click the `Lock` icon, which represents the SSL/TLS status.
+
This will indicate information about your current connection.
```